PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -190,7 +221,13 @@ for STRING in "$AtlasSpaceFolder"/"$NativeFolder"@native@roi "$AtlasSpaceFolder"
   Folder=`echo $STRING | cut -d "@" -f 1`
   Mesh=`echo $STRING | cut -d "@" -f 2`
   ROI=`echo $STRING | cut -d "@" -f 3`
-  for STRINGII in MyelinMap@func SmoothedMyelinMap@func MyelinMap_BC@func SmoothedMyelinMap_BC@func corrThickness@shape ; do
+
+  STRINGIIList="corrThickness@shape"
+  if [ "${T2wPresent}" = "YES" ] ; then
+    STRINGIIList+=" MyelinMap@func SmoothedMyelinMap@func MyelinMap_BC@func SmoothedMyelinMap_BC@func"
+  fi
+
+  for STRINGII in $STRINGIIList ; do
```

Comment:
Is there really a need to define a `STRINGIIList` variable here?  Isn't it always going to be equal to the previously defined `STRINGList`?    If so, then all we need is the single line:
`for STRINGII in $STRINGList; do`

Reply:
Yep, and we don't need to have it within the loop, so I put it before the loop.